### ISSUE TITLE
Assign to same false negative

### ISSUE THREAD

**Issue by JohanLarsson** [JohanLarsson/Gu.Analyzers#5]
```C#
foo1.Bar = foo2.Bar;
```
